src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17283.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if ANDROID
```

Comment:
Does it make sense to run this test on iOS and WinUI as well?

We typically let these tests run on all platforms even if the original issue is only happening on Android. This also lets us make sure that we have consistent behavior between all platforms.

If it's broken on iOS and/or WinUI then log a bug and make a note of the bug inside this test.

Reply:
Okay, I have modified the script to run the test on both iOS and WinUI. Could you please take a look and let me know?